include/class.mail.php

```diff
@@ -326,6 +326,14 @@ public function readLine() {
              return trim($result);
          }
 
+         public function login($user, $password, $tryApop = true) {
+             try {
+                parent::login($user, $password, $tryApop);
+                return true;
+             } catch (\Throwable $e) {
+                 throw new Exception(__('cannot login, user or password wrong'));
```

Comment:
Thanks for the find & fix 

I think we should  sarface whatever the error the mail server / backend kicked back. 

```php 
throw new Exception($e->getMessage());
```

Also curious why you're using POP instead IMAP.  Not judging ;)

Reply:
Yes you are right. I changed the exception message including the responded error...
I don't like POP, but a client of me need it. He has an Office365 mail without IMAP enabled. He says, it's a cheaper license.
But there is also the possibility to use POP, then that should work too.